src/internal/translations/en-US.json

```diff
@@ -399,6 +399,46 @@
         "comment": "",
         "limit": 0
     },
+    "properties.axis.name": {
+        "value": "name",
+        "comment": "Name of Highcharts axis",
+        "limit": 0
+    },
+    "properties.axis.name.position": {
+        "value": "Position",
+        "comment": "Position of Highcharts axis",
+        "limit": 0
+    },
+    "properties.axis.name.position.left": {
+        "value": "left",
+        "comment": "",
+        "limit": 0
+    },
+    "properties.axis.name.position.center": {
+        "value": "center",
+        "comment": "",
+        "limit": 0
+    },
+    "properties.axis.name.position.right": {
+        "value": "right",
+        "comment": "",
+        "limit": 0
+    },
+    "properties.axis.name.position.top": {
+        "value": "top",
+        "comment": "",
+        "limit": 0
+    },
+    "properties.axis.name.position.middle": {
+        "value": "middle",
```

Comment:
Vertical center (between top and bottom)

Reply:
Fixed